db/drivers/postgres/grass-pg.md

```diff
@@ -115,8 +115,8 @@ v.info -t test
 
 #### Geometry Converters
 
-- [PostGIS with
-  shp2pgsql](https://postgis.net/workshops/postgis-intro/loading_data.html#loading-with-shp2pgsql):  
+- [PostGIS with shp2pgsql]
+  (<https://postgis.net/workshops/postgis-intro/loading_data.html#loading-with-shp2pgsql>):
```

Comment:
This isn't working, at least for GitHub's markdown rendering. The square brackets and parentheses are separated by a space instead of being considered a link